client/src/Main.elm

```diff
@@ -1006,10 +894,10 @@ visualRepresentation model =
             sortByWith .bsa1_overhang ascending insertRecordList
 
         chartLabels =
-            List.reverse (model.selectedBackbone.name :: List.reverse (List.map .name sortedInsertRecordList))
+            List.reverse ((Maybe.withDefault "" <| Maybe.map .name model.selectedBackbone) :: List.reverse (List.map .name sortedInsertRecordList))
```

Comment:
Little nervous about 2x `List.reverse` here.

Reply:
This seemed like a good way to add the backbone to the beginning of the list.
Is there a better way?

Reply:
The final `reverse` puts it at the end though. Actually now I'm not sure why you're reversing at all